srtcore/srt.h

```diff
@@ -358,6 +363,11 @@ struct CBytePerfMon
    uint64_t byteSndDrop;                // number of too-late-to-send dropped bytes
    uint64_t byteRcvDrop;                // number of too-late-to play missing bytes (estimate based on average packet size)
    uint64_t byteRcvUndecrypt;           // number of undecrypted bytes
+
+   uint32_t countBreak;                 // increased with every broken link in the group
+   uint32_t countActivate;              // increased with every activated link in the group
+   uint32_t countEager;                 // increased with every case when unstable link became stable again
```

Comment:
Recovered or restored might be better. 🤔 
```suggestion
   uint32_t countRecovered;                 // increased with every case when unstable link became stable again
```

Reply:
The motivation for this field is to count the cases when a link has been considered unstable, which has forced the group sender to activate the backup link, but then it turned out that the link actually wasn't broken and has returned to stability. This means that the activation was based on a false alert and resulted in only unnecessary overhead. A user who is reviewing these statistics, when seeing any nonzero value here, and moreover widely exceeding the value from `countBroken`, it should be an information for him that he definitely has too small value of `groupstabtimeo` option.

Hard to fit such an explanation in one comment line.

Reply:
Some better names I could propose would be:

* countFalseAlert
* countMissed
* countExcess